src/sentry/search/eap/resolver.py

```diff
@@ -468,7 +471,7 @@ def resolve_aggregate_term(
 
     def _resolve_search_value(
         self,
-        column: ResolvedColumn,
+        column: ResolvedColumn | ResolvedFormula,
```

Comment:
Another one, is this intentional? We don't resolve search values for formulas do we?